collections/blueprints/pci.md

```diff
@@ -0,0 +1,268 @@
+    </BasisTheoryProvider>
+  );
+}
+```
+
+Next, add the following code to your new Card Form component:
+
+```jsx
+import { 
+  useBasisTheory
+} from '@basis-theory/basis-theory-react';
+
+export const CardForm = () => {
+  const { bt } = useBasisTheory();
+
+  const submit = async () => {};
+
+  return (
+    <form onSubmit={submit} className="form">
+      <button type="button" onClick={submit} >
+        Submit
+      </button>
+    </form>
+  );
+}
+```
+
+This will initialize Basis Theory JS and add your new card form.
+
+### Add Card Element
+
+Now, we need to add a [`CardElement`](https://docs.basistheory.com/elements/#element-types-card-element) component to our form. This Element type offers a single line to capture the card number, expiration date, and CVC.
+
+Add the `CardElement` to our inputs:
+
+```jsx
+import { 
+  useBasisTheory,
+  CardElement
+} from '@basis-theory/basis-theory-react';
+```
+
+And inside of our `<form>` tag, add the following code:
+
+```jsx
+<CardElement id="card" />
+```
+
+Your card component should now look like this:
+
+```jsx
+import { 
+  useBasisTheory,
+  CardElement
+} from '@basis-theory/basis-theory-react';
+
+export const CardForm = () => {
+  const { bt } = useBasisTheory();
+
+  const submit = async () => {};
+
+  return (
+    <form onSubmit={submit} className="form">
+      <CardElement id="card" />
+
+      <button type="button" onClick={submit} >
+        Submit
+      </button>
+    </form>
+  );
+}
+```
+
+You can fully customize the [look and feel](./guides/secure/../../../guides/secure/style-elements-for-my-brand.md) to match your existing site and even listen to common [events](https://docs.basistheory.com/elements/#element-events) on all Elements components.
+
+## Store Cards
+
+Now that we have securely captured the card information, we need to securely store the card. 
+
+To do this, we will tokenize the data with Basis Theory. Basis Theory handles all of the secure encryption and storage of the cardholder data and returns a non-sensitive token identifier that can be stored in your database.
+
+### Tokenize the Card
+
+Inside of your `CardForm`, we will update the `submit` function to tokenize the card using the PCI-compliant [`card` Token Type](https://docs.basistheory.com/#token-types-card):
+
+```jsx
+const submit = async () => {
+  // Tokenize card with Basis Theory
+  const token = await bt.tokenize({
+    type: 'card',
+    data: bt.getElement('card'),
+  });
+
+  // Submit card token to API
+  // Example using Axios
+  const { data } = await axios.post('/api/cards', { cardTokenId: token.id });
+};
+```
+
+### What is Happening?
+
+When a user submits their payment information, we will tokenize the underlying value of the [CardElement](https://docs.basistheory.com/elements/#cardelement). This will instruct Basis Theory Elements to submit directly to [Basis Theory's Tokenize endpoint](https://docs.basistheory.com/#tokenize) and return the resulting token identifiers.
```

Comment:
```suggestion
When a user submits their payment information, we will tokenize the underlying value of the [CardElement](https://docs.basistheory.com/elements/#cardelement). This will instruct Basis Theory Elements to submit this sensitive card data directly to [Basis Theory's Tokenize endpoint](https://docs.basistheory.com/#tokenize) and return the resulting token identifiers to our front end, all without our application having accessed the sensitive card data directly.
```